_data/projects/Secluded.yml

```
@@ -0,0 +1,13 @@
+name: Secluded
+desc: Secluded is a webpage that tries to be isolated from web crawers although publically visible. It is a crawler behaviour experiment
```

Did you meant?
```suggestion
desc: Secluded is a webpage that tries to be isolated from web crawlers although publically visible. It is a crawler behaviour experiment
```
or is this intentional?

---

Please correct if this is a typo @bacloud14

---

yes you are right @ritwik12 
thank you so much

---

Welcome :)